lib/kino/utils/table.ex

```diff
@@ -0,0 +1,121 @@
+defmodule Kino.Utils.Table do
```

Comment:
Maybe Kino.Table.Utils? It feels Table is more important but that’s a nitpick. :)

Reply:
My reasoning is that `Utils` is less specific and can thus serve as a namespace, no strong opinion though.

Reply:
Fine by me :)